topo_processor/cli/main.py

```diff
@@ -45,38 +45,31 @@ def wrapper(*args, **kwargs):
     required=True,
     help="The target directory path or bucket name of the upload",
 )
-@click.option(
-    "-u",
-    "--upload",
-    is_flag=True,
-    help="If True will be uploaded to the specified target s3 bucket otherwise will be stored locally in specified target location",
-)
 @click.option(
     "-v",
     "--verbose",
     is_flag=True,
     help="Use verbose to display trace logs",
 )
 @coroutine
-async def main(source, datatype, target, upload, verbose):
+async def main(source, datatype, target, verbose):
     if verbose:
         set_level(LogLevel.trace)
     start_time = time_in_ms()
-    source_dir = os.path.abspath(source)
     data_type = DataType(datatype)
+
+    source_dir = os.path.abspath(source)
     await process_directory(source_dir)
+
+    file_system = get_file_system(target)
```

Comment:
we need to check the source and target, the source can be remote / local; the target can be remove / local